src/render/utf8.rs

```diff
@@ -0,0 +1,110 @@
+//! UTF-8 rendering, with 2 pixels per symbol.
+
+use crate::render::{Canvas as RenderCanvas, Pixel, Color};
+
+const CODEPAGE: [&str; 4] = [" ","\u{2584}","\u{2580}","\u{2588}"];
+
+#[derive(Copy, Clone, PartialEq)]
+pub enum Unicode1x2 {
+    Dark, Light
+}
+
+impl Pixel for Unicode1x2 {
+    type Image = String;
+    type Canvas = Canvas;
+    fn default_color(color: Color) -> Unicode1x2 { color.select(Unicode1x2::Dark, Unicode1x2::Light) }
+    fn default_unit_size() -> (u32, u32) { (1, 1) }
+}
+
+impl Unicode1x2 {
+    fn value(&self) -> u8 {
+        match self {
+            Unicode1x2::Dark => {1}
+            Unicode1x2::Light => {0}
+        }
+    }
+    #[doc(hidden)]
+    fn parse_2_bits(sym: &u8) -> &'static str {
+        CODEPAGE[*sym as usize]
+    }
+}
+
+pub struct Canvas {
```

Comment:
```suggestion
pub struct Canvas1x2 {
```

Reply:
It should be easy to change chunks to make it 2x3 or whatever, so maybe make it universal up to 8 bits per symbol?

Reply:
Speaking of that, I can replace original string module with this on 1-bit encoding.

Reply:
> It should be easy to change chunks to make it 2x3 or whatever

@cab404 the implementation seems specific to 1x2 though.

Reply:
It needs minor changes — ability to change codepage, and a bit different zipping to accomodate >2 rows. But okay, let's merge this first, I'll change it)